test/index.test.js

```diff
@@ -124,7 +124,7 @@ describe('config parser', () => {
                 () => parser(loadData('bad-step-name.yaml'))
                     .then((data) => {
                         assert.match(data.jobs.main[0].commands[0].command,
-                            /"foo bar" only supports the following characters A-Z,a-z,0-9,-,_/);
+                            /foo bar" only supports the following characters A-Z,a-z,0-9,-,_/);
```

Comment:
is this weird? why is it missing a leading quotation mark?

Reply:
The complete message is actually [Validation Error] "foo bar" ..., so we are matching the pattern.

Reply:
shouldn't it have matched with the `"`?

Reply:
Actual error message now includes the entire object stucture.
ValidationError: "jobs.main.steps[0].foo bar" only supports the following characters A-Z,a-z,0-9,-,_
So made a limited change to message.

Reply:
missed that